### PR TITLE
Add Nlp service to CI

### PR DESCRIPTION
**Pull request by fdore**

### CODE REVIEW

File: src/docker-compose.yml [hunk 1]
```diff
@@ -83,3 +83,15 @@ services:
     environment:
      - BROKER_HOST=broker
      - BROKER_PORT=8000
+  nlp:
+    build:
+      dockerfile: Nlp.Dockerfile
```
Comment (bhavesh-desai-scratch) [inline src/docker-compose.yml]: Is the casing right here?

**Comment by bhavesh-desai-scratch** [comment]
You need to create some k8 files as well to get it into enviroments

File: src/docker-compose.yml (hunk 1 above)
Comment (fdore) [reply src/docker-compose.yml]: yes

**Comment by fdore** [comment]
> You need to create some k8 files as well to get it into enviroments

Forgot to run it from the circle ci config

**Pull request merged**